packages/itwinui-css/src/table/classes.scss

```diff
@@ -30,34 +27,38 @@
   @include iui-table-body;
 }
 
-.iui-cell {
+.iui-table-cell {
   @include iui-table-cell;
 }
 
-.iui-cell-shadow-left {
+.iui-table-cell-shadow-left {
   @include iui-table-cell-shadow-left;
 }
 
-.iui-cell-shadow-right {
+.iui-table-cell-shadow-right {
   @include iui-table-cell-shadow-right;
 }
 
-.iui-paginator {
-  @include iui-paginator;
+.iui-table-paginator {
```

Comment:
I don't remember whether we intentianally named it without prefix. Maybe @FlyersPh9 you remember?

Reply:
I suppose we could have a paginator on a page with tiles, but I think it might look out of place with the darker background color that it currently has.

Reply:
From your previous comment

> Maybe we should prefix all elements but not modifiers.

I figured `iui-paginator` was an element, so I prefixed it with `iui-table`. I assume when you mean "modifiers" you are talking about things like `iui-slot` and `iui-sorted` which I remained untouched